eks-distro-base/Dockerfile.minimal-base-nsenter

```diff
@@ -0,0 +1,32 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# *NOTE* we have to limit our number of layers heres because in presubmits there
+# is no overlay fs and we will run out of space quickly
+
+################# BASE-NONROOT ################
```

Comment:
Super tiny nit and I suspect we have this issue in other files, but change this to base!-nsenter instead of base-nonroot

Reply:
Good catch! I have actually changed it to `## BUILDER ##`  to be consistent with most of the other Dockerfiles. The `csi-ebs` one is the one which has the same problem at this point. I didn't fix that one in this PR to not pollute its scope.